recipes/zeromq/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class ZeroMQConan(ConanFile):
+    name = "zeromq"
+    homepage = "https://github.com/zeromq/libzmq"
+    description = "ZeroMQ is a community of projects focused on decentralized messaging and computing"
+    topics = ("conan", "zmq", "libzmq", "message-queue", "asynchronous")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "LGPL-3.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "encryption": [None, "libsodium", "tweetnacl"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "encryption": "libsodium",
+    }
+    generators = "cmake"
```

Comment:
The recipe contains a `find_package(Sodium)` to look for libsodium. We should add the `cmake_find_package` generator so it uses Conan generated `FindXXXX.cmake` file.

Reply:
I addressed this change request.